admin/tools/docker/deployment/localhost/run-multiczar-tests.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/sh
+
+# @author  Fabrice Jammes, IN2P3/SLAC
```

Comment:
Did Fabrice wrote this?
I think we should drop all `@author` strings from code, git history is better for that purpose.